source/camera/basic/CaptureTutorial.md

```diff
@@ -47,7 +46,7 @@ camera = app.connect_camera(serial_number="2020C0DE")
 ---
 **Note** 
 
-The serial number of your camera is shown in the Zivid Studio.
+The serial number of your camera is shown in Zivid Studio.
```

Comment:
should this be part of a different commit?

Reply:
maybe, but I wouldn't want a separate commit for it in the master history. It will be squashed after PR anyway, so maybe just leave it?